### DISCUSSION TITLE
Green Line along top of RGB Matrix

### DISCUSSION THREAD

**Discussion by burreyein** [riffnshred/nhl-led-scoreboard#453] [General]
Having an issue with a solid green line coming across the top pixels of my 32x64..

It isn't happening on every page..  Loading screen, the top line is completely blank.  When I see Colorado vs. St Louis tonight, the top line is as expected - no green at all.  As soon as it bumps to the next game or team standings page, the solid green line across the top is back.

I tried to update to 1.8 but it disabled my display completely, so I'm back on a fresh install of 1.7.0 with an updated config.json.

Currently powered via 5.0v/4.0a power supply to the AdaFruit bonnet..  Seeing an Undervoltage Warning when watching the script, so I tried adding a power supply to the Pi (5.0v/0.4a) with no change in results.  I have a 5.0v/2.0a for the Pi on the way from Amazon set to deliver tomorrow hoping that might make a difference, (though I'm certainly less than confident.)  Went ahead and posted in case somebody has the same issue, as I haven't been able to find much help trolling through GitHub.  Will verify whether or not it works, but would like some insight if nothing else.

Please advise, thanks for keeping this project relevant!!

**Comment by azaroth08** [comment]
Run a fresh install of 1.8.x.   Don't try and update, freshly install.  If the img version isnt 1.8.2 youll have to run an update from there also. I'm not sure it's recommended to add a second power supply to the pi while running the bonnet, and the current supply should be more than enough.

**Comment by falkyre** [comment]
What's your command line for running the scoreboard?  Does it contain an --updatecheck in it (if you are on the image it will).  If so, that green line means there's an update and it only shows on certain boards.  It's likely not an issue with power supply.  The green line will show on weather, weather alert, team summary, standings and clock boards

**Comment by burreyein** [comment]
Both of you were right - I didn't bother with the power supply..

Verified that --updatecheck was part of the command line, but I'm pretty amateur and had trouble finding scoreboard.conf to edit.
Fresh install of 1.8.2 working like a champ.

Thank you both!
